pkg/controller/manage/manage_controller.go

```diff
@@ -64,6 +68,23 @@ type KmeshManageController struct {
 	client            kubernetes.Interface
 }
 
+type podEventArgs struct {
```

Comment:
donot do this kind of refactor in this bug fix, otherwise it is a little bit hard to review

Reply:
done, will be edited in next PR